source/server/listener_manager_impl.cc

```diff
@@ -1018,6 +1021,9 @@ Network::DrainableFilterChainSharedPtr ListenerFilterChainFactoryBuilder::buildF
                                      "{}. \nUse QuicDownstreamTransport instead.",
                                      transport_socket.DebugString()));
   }
+#else
+  UNREFERENCED_PARAMETER(listener_);
```

Comment:
can is_quic be true if it's compiled out (do we need error handling here) or would we have failed out on an earlier path?

Reply:
I believe we should have failed out earlier, but I will confirm and add more comments and/or TODOs